lib/schedule_events.js

```diff
@@ -12,7 +12,11 @@ const ScheduleEvents = function(aws) {
 
 ScheduleEvents.prototype = {
   _ruleDescription: (params) => {
-    return `${params.ScheduleName} - ${params.ScheduleExpression}`;
+    if (params.ScheduleDescription != null) {
```

Comment:
this would also mean that this would have to check if `ScheduleDescription` is in `params`, and `not null`